articles/microsites/call-api/call-api-single-page-app.md

```diff
@@ -15,16 +15,16 @@ useCase:
 
 Using Auth0 in your applications means that you will be "outsourcing" the authentication process to a centralized login page in the same way that Gmail, YouTube, and any other Google property redirects to [accounts.google.com](http://accounts.google.com) whenever a user signs in.
 
-Your user will authenticate, and Auth0 will generate an ID Token and Access Token that will be passed back to your application. The Access Token can then be used to call your API and extract attributes for that user (such as name, email, role, or a custom attribute)
+Your user will authenticate, and Auth0 will generate an ID Token and Access Token that will be passed back to your application. The Access Token can then be used to call your API.
 
 ## How it works
 
 When your app needs to fetch user data from your API:
 
-1. If the user is not already authenticated, our SDK redirects the user to your Auth0 Authorization Server.
-2. The user authenticates with Auth0 using one of your configured login options (e.g., username/password, social identity provider, SAML).
-3. Your app requests an ID Token, Access Token, and Refresh token.
-4. Auth0 responds with the requested tokens.
+1. If the user is not already authenticated, our SDK redirects the user to your Auth0 Authorization Server, requesting an Authorization Code and ID Token.
+2. The user authenticates with Auth0 using one of your configured login options (e.g., username/password, social identity provider, SAML), and Auth0 responds with the code and ID Token.
+3. Your app's API back-end can use the code to request an ID Token, Access Token, and Refresh Token from Auth0's Token endpoint.
```

Comment:
This seems far more detailed than any other microsite flow, which is supposed to be very basic. We detail every step of this flow on the next page down (the description of the hybrid flow). Are we perhaps getting a little too detailed for this level with all the code and token endpoint details? (Again, checking that this is a conscious choice to be an outlier.)

Reply:
Also, as we discussed when tightening up the Hybrid Flow docs, we're specifically pushing the Hybrid Flow as the majority case scenario when calling an API. Is that not the case with the main microsite doc?